src/main/java/com/coveros/selenified/element/Get.java

```diff
@@ -295,8 +293,7 @@ public int numOfSelectOptions() {
      */
     public String[] selectOptions() {
         if (!isPresentSelect()) {
-            return null; // NOSONAR - returning an empty array could be confused
-            // with no options available
+            return null; // returning an empty array could be confused with no options available
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Return an empty array instead of null. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1168)